### PR TITLE
Tickets/sitcom 428

### PR DESCRIPTION
**Pull request by isotuela**
Hi Bruno, 
If you could please take a look at this LVV-2219, I'd appreciate it. I look forward to hearing your comments! 

Thank you, 
ioana

### CODE REVIEW

**Comment by b1quint** [comment]
Hi Ioana, could you clean up the outputs, please? Here are a few other suggestions:
- Rebase this branch using `develop`. This will bring some extra functionality that you can use. 
- Instead of using the block of code in the cell [2], use `vandv.info.ExecutionInfo`. 
- When doing imports, put the standard python imports first organized alphabetically followed by the lsst imports organized alphabetically. [LVV-T1802](https://github.com/lsst-sitcom/notebooks_vandv/blob/develop/notebooks/proj_sys_eng/sitcom_integration/LVV-T1802-Integ_m2hex_sal.ipynb) can be used as an example (and I have to clean its outputs as well). 
- Create the efd client using `client = vandv.efd.create_efd_client()`
- Instead of having "Step 1", "Step 2", etc, use some title associated with the test step or group of test steps. The reason for this is because we might add extra steps and then we will have extra work updating these numbers for all the test cases. For example, your current "Step 1" marks the beginning of starting up ComCam. Put `ComCam StartUp` as a level-2 title and the following steps as standard cells. Another example, Steps 7 to 9 are all part of `Image Test`.

**Comment by isotuela** [comment]
Waiting on Kevin's update of the Jira test case with XML changes.

**Review by b1quint — CHANGES_REQUESTED** [review]
Hi Ioana, 
Could you implement the following changes?
- Clear notebook output
- Use the `vandv.get_index` function to retrieve the Script index. This function uses the test case's last four digits plus the month and day to determine the index. Also, I talked to Tiago, and we agreed that indexes for test cases should be negative. The final format is `-TTTTMMDD`.
- (suggestion) When having code in the middle of the markdown text, use a bold or fixed-width font to increase readability. Same when you have some warning message (like the "beware" in step 7. 
- Do you have a link describing the substates in the different steps? I believe it would be helpful to add it in the beginning so people can quickly understand what is happening. A link in the header of the notebook should be sufficient. 
- Step 24 - I guess there is a `set_start` missing after `comcam.rem.cccamera.cmd_stop`?
- I suggest adding a note at the beginning saying that when `check_telemetry` raises an error, it means that the telemetry is not available in the EFD. Or, add some exception handling with a message stating that. 
-

File: notebooks/proj_sys_eng/sitcom_integration/LVV-T2219-ComCam_Interface_to_SAL.ipynb [hunk 1]
```diff
@@ -2194,27 +1272,16 @@
...
-     },
-     "execution_count": 80,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "comcam.rem.cccamera.cmd_stop"
+    "comcam.rem.cccamera.cmd_stop.set_start()"
```
Comment (b1quint) [inline notebooks/proj_sys_eng/sitcom_integration/LVV-T2219-ComCam_Interface_to_SAL.ipynb:1847]: I guess this requires an `await`?

**Review by b1quint — CHANGES_REQUESTED** [review]
In general, all good! 
There is a single `await` which I believe it is missing on step 24.

File: notebooks/proj_sys_eng/sitcom_integration/LVV-T2219-ComCam_Interface_to_SAL.ipynb (hunk 1 above)
Comment (isotuela) [reply notebooks/proj_sys_eng/sitcom_integration/LVV-T2219-ComCam_Interface_to_SAL.ipynb:1847]: Thank you for your careful attention. `await comcam.rem.cccamera.cmd_stop.set_start()` gives an error, whereas the same command without the `await` doesn't when run in the TTS, it's been decided to leave it without the `await` until we run it in the summit. It might be that the command is not yet implemented, as it was mentioned in conversations with Kevin and Tony.

**Review by b1quint — APPROVED** [review]
We are good now. @isotuela explained that the `cmd_stop` command is not implemented yet via Slack. We might need to revisit this in the future. For now, we are good to go!
